engine/apps/alerts/migrations/0010_channelfilter_filtering_term_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.17 on 2023-03-06 07:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('alerts', '0009_alertreceivechannel_web_templates_modified_at'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='channelfilter',
+            name='filtering_term_type',
+            field=models.IntegerField(choices=[(0, 'regex'), (1, 'jinja2')], default=None, null=True),
```

Comment:
Can we have this field non-nullable, but with default value 0?

Reply:
I think it's valid tradeoff to experience 500's to have more consistent database and codebase. With nullable we need a lot of code to show this field in private/public API correctly and to check for null to write to that field from API.